### PR TITLE
Add locksmith build

### PR DESCRIPTION
**Pull request by aevox**
Add locksmith application and docker image build

### CODE REVIEW

File: pkgs/vault-fernet-locksmith/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  goPackagePath = "github.com/aevox/vault-fernet-locksmith";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "aevox";
+    repo = "vault-fernet-locksmith";
+    sha256 = "0q8sl6nd956hsh7bdnblydy1n2ql7fl9jcrl3zxgbcwb50rz293a";
+  };
+
+  goDeps = ./deps.nix;
```
Comment (nlewo) [inline pkgs/vault-fernet-locksmith/default.nix]: Can you remove this attribute since it is empty?

File: pkgs/vault-fernet-locksmith/deps.nix [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
```
Comment (nlewo) [inline pkgs/vault-fernet-locksmith/deps.nix]: This file is useless

Comment (aevox) [reply pkgs/vault-fernet-locksmith/deps.nix]: indeed !

File: pkgs/vault-fernet-locksmith/default.nix (hunk 1 above)
Comment (aevox) [reply pkgs/vault-fernet-locksmith/default.nix]: done

**Pull request merged**